### ISSUE TITLE
Can I use this code to get other dataset masks

### ISSUE THREAD

**Issue by lsj1111** [ViTAE-Transformer/SAMRS#24]
it's a good job，and I saw in your paper that you obtained the segmentation results of DOTA 2.0 version. Can I use this code to obtain the segmentation results of DOTA 1.0?
look forward to you reply,
thank you very much

**Comment by DotWang** [comment]
@lsj1111 Of course, we have released the code. As I know, DOTA-v1.0 is actually a subset of DOTA-v2.0 and they have the same label format, thus it is easy to use our code for performing the label conversion.

**Comment by DotWang** [comment]
@lsj1111 Even if for other detection datasets, our code is still available, but you may need to manually change the code of reading detection labels according to corresponding formats.

**Comment by lsj1111** [comment]
> @lsj1111 Even if for other detection datasets, our code is still available, but you may need to manually change the code of reading detection labels according to corresponding formats.

OK, I got it ,thank you very much

**Comment by lsj1111** [comment]
> @lsj1111 Even if for other detection datasets, our code is still available, but you may need to manually change the code of reading detection labels according to corresponding formats.

When I changed the data set to dota1.0, there were more than 20,000 pictures in total, but the output segmentation results were only more than 13,000. What is the problem?According to the printed information, it can be seen that there is indeed a problem during the segmentation process：
dota dataset contains 21046 images

**Comment by DotWang** [comment]
@lsj1111 The segmentaation map cannot be produced if there is no box

**Comment by lsj1111** [comment]
> @lsj1111 The segmentaation map cannot be produced if there is no box, some samples in DOTA 1.0 have not been annotated.

yes,you are right,i got a new problem, for the same picture, I used DOTA_devkit to crop the picture. For detection tasks, the annotation when cropping is usually Rbox, but in your paper I found that using hbox as a prompt when predicting the mask will be better, but these two annotations The Ground Truth results obtained by different methods are different.
![e64f775dab05dd9b1abe13e3b1586f4](https://github.com/user-attachments/assets/8b09db65-18b0-42f6-9e97-4e541a777f19)
左边是Hbox注释得到的结果，右边是Rbox得到的结果

**Comment by DotWang** [comment]
@lsj1111 For R-Box, you should use RH-Box as the prompt
